apps/backend/src/purchases/purchases.controller.ts

```diff
@@ -82,7 +82,7 @@ export class PurchasesController {
   @ApiOkResponse({ schema: purchaseEventsSchema })
   @ApiParam({ name: 'id', type: String })
   async getBlockchainEvents(@Param('id') id: string) {
-    return this.purchasesService.getChainEvents(id);
+    return []; // this.purchasesService.getChainEvents(id); TODO: RE-ENABLE 
```

Comment:
why can't we have it now ?

Reply:
Wanted to implement this in a separate PR, so that this PR doesn't get even bigger.